test/job.js

```diff
@@ -46,13 +48,82 @@ describe('/POST Post a new job', () => {
           response.body.should.have
             .property('message')
             .eql('Create the job successful!');
-          response.body.job.should.have.property('id');
-          response.body.job.should.have
-            .property('title')
-            .eql(jobParameters.title);
-          response.body.job.should.have
-            .property('tags')
-            .eql(jobParameters.tags);
+          id = response.body.job.id;
+          done();
+        });
+    });
+  });
+
+  describe('/PUT Update the job', () => {
+    it('Update the job with properly parameters', (done) => {
```

Comment:
The title of the test case should descript by this pattern "when ... then" or "return ... when". For example:
`it('return status 200 when update successfully')`

However, it's not only the status response from the API but the response object as well.

We have to make sure that after the update successfully, we return the correct job with the latest information